skippkg-finder.py

```diff
@@ -0,0 +1,333 @@
+#!/usr/bin/python3
+
+import argparse
+import logging
+import sys
+
+from urllib.error import HTTPError
+
+import re
+from lxml import etree as ET
+from collections import namedtuple
+
+import osc.conf
+import osc.core
+from osc import oscerr
+from osc.util.helper import decode_it
+from osclib.core import source_file_ensure
+
+OPENSUSE = 'openSUSE:Leap:15.4'
+SLE = 'SUSE:SLE-15-SP4:GA'
+SUPPORTED_ARCHS = ['x86_64', 'i586', 'aarch64', 'ppc64le', 's390x']
+DEFAULT_REPOSITORY = 'standard'
+BINARY_REGEX = r'(?:.*::)?(?P<filename>(?P<name>.*)-(?P<version>[^-]+)-(?P<release>[^-]+)\.(?P<arch>[^-\.]+))'
```

Comment:
as they are regexp why not `BINARY_REGEX = re.compile(r'...` and some comment explaining regexp would be nice as regexps are kind of tricky.

Reply:
I didn't bother to create another one, just copy it from https://github.com/openSUSE/openSUSE-release-tools/blob/master/osclib/core.py#L36 , I need `<filename>` only though.

Reply:
well, if it is copy, maybe you can use it directly. either call `osclib.core.BINARY_REGEX` or if you want to import it then `from osclib.core import BINARY_REGEX` works in a same way and as bonus if regex change to match newer binaries, your code start using it.